[2025-09-11T10:28:00.731+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-11T10:28:00.768+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_hourly_predictions.make_prediction manual__2025-09-11T10:27:56.236225+00:00 [queued]>
[2025-09-11T10:28:00.781+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_hourly_predictions.make_prediction manual__2025-09-11T10:27:56.236225+00:00 [queued]>
[2025-09-11T10:28:00.783+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-11T10:28:00.803+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): make_prediction> on 2025-09-11 10:27:56.236225+00:00
[2025-09-11T10:28:00.819+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_hourly_predictions', 'make_prediction', 'manual__2025-09-11T10:27:56.236225+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/prediction_dag.py', '--cfg-path', '/tmp/tmpirtg5ux6']
[2025-09-11T10:28:00.823+0000] {standard_task_runner.py:91} INFO - Job 39: Subtask make_prediction
[2025-09-11T10:28:00.823+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=3020) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-11T10:28:00.826+0000] {standard_task_runner.py:64} INFO - Started process 3022 to run task
[2025-09-11T10:28:00.914+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_hourly_predictions.make_prediction manual__2025-09-11T10:27:56.236225+00:00 [running]> on host 94617eeb311b
[2025-09-11T10:28:01.082+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ops-team' AIRFLOW_CTX_DAG_ID='simple_hourly_predictions' AIRFLOW_CTX_TASK_ID='make_prediction' AIRFLOW_CTX_EXECUTION_DATE='2025-09-11T10:27:56.236225+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-11T10:27:56.236225+00:00'
[2025-09-11T10:28:01.084+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-11T10:28:01.112+0000] {logging_mixin.py:188} INFO - üîÆ Making delay prediction...
[2025-09-11T10:28:27.133+0000] {logging_mixin.py:188} INFO - ‚ùå Not enough recent data
[2025-09-11T10:28:27.300+0000] {python.py:237} INFO - Done. Returned value was: insufficient_data
[2025-09-11T10:28:27.301+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-11T10:28:27.329+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_hourly_predictions, task_id=make_prediction, run_id=manual__2025-09-11T10:27:56.236225+00:00, execution_date=20250911T102756, start_date=20250911T102800, end_date=20250911T102827
[2025-09-11T10:28:27.432+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-11T10:28:27.456+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-11T10:28:27.460+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
